scripts/check_python_format.sh

```diff
@@ -5,6 +5,10 @@ ISORT="${ISORT:-isort}"
 
 set -eu
 
+# Know the specific versions used to reproduce the formatting checks locally
+${BLACK} --version
+${ISORT} --version
```

Comment:
I didn't understand why this was required and/or why this is a part of this PR

Reply:
I found myself again in a situation where `scripts/check_python_format.sh` says the python input file is formatted ok, but when I commit the python file to CI, CI says it needs reformatting. This happens because there are different versions of `black` and `isort` on my local laptop and on CI runners, while before this change there was no way of figuring out what exact versions are used in the CI runners. This change prints the versions to the output that is then shown in CI logs. After I got the versions from CI, I installed the exact same versions locally, ran the formatting script, it changed the python file and only then that python file was accepted by CI.

Reply:
And thanks for the clarification! Please include some context like that in the commit message.

Reply:
@srogatch Sorry about this. I have known about this issue for a long time and failed to fix it. This is a decent hack to help for the moment, until I can create a better fix that actually checks the version and fails out if it's wrong. Thanks. (https://katanagraph.atlassian.net/browse/KAT-1287)